src/styles/abstracts/_variables.scss

```diff
@@ -3,8 +3,18 @@ $color-gray-bg: #eee;
 $color-lightgray-font: #bdbdbd;
 $color-darkgray: #333;
 $color-black: #000;
+$color-yellow-1: #ffecb7;
+$color-yellow-2: #eedaa3;
+$color-brown-font: #551b05;
+$color-brown-font-disabled: #551c06;
+$color-green-disabled: #8dad84;
+$color-green-main: #06836d;
+$color-green-home-bg: #067c67;
+$color-green-dark-1: #004636;
+$color-green-dark-2: #00311e;
```

Comment:
are those colors in any way related? for example is `$color-green-dark-1` the darker version of `$color-green-main`?
if so I would consider defining main colors only and calculating further colors based on these main colors
sass can manipulate colors: https://sass-lang.com/documentation/modules/color
then it's easier to tune/update/replace color theme

Reply:
I don't know 🤷🏻‍♀️ I got them from a friend